pipelines/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md

```diff
@@ -1,3 +1,13 @@
+# smartseq2_v4.0.0
+
+2020-06-04 (Date of Last Commit)
+
+* Added loom output and removed the zarr output
+
+* Loom file attribute names have changed: CellID: cell_names, Gene: gene_names and Accession: ensembl_ids
+
+* Loom file name has changed from out.loom to "sample_id".loom
+
```

Comment:
We should include the addition of the expected counts in the loom output here

Reply:
Done.